libs/api-client-bear/src/user.ts

```diff
@@ -2,7 +2,18 @@
 import qs from "qs";
 import { XhrModule, ApiResponse } from "./xhr.js";
 import { ProjectModule } from "./project.js";
-import { GdcUser, GdcOrganization } from "@gooddata/api-model-bear";
+import {
+    IAccountInfo,
+    IAccountInfoResponse,
+    IAccountSetting,
+    IBootstrapResource,
+    ISeparators,
+    ISeparatorsResponse,
+    IWrappedAccountSetting,
+    IFeatureFlags as IBearFeatureFlags,
```

Comment:
Looks like the exactly the same type as IFeatureFlags, we should use only one of them and remove the later one.

Reply:
Fixed